.eslintignore

```diff
@@ -0,0 +1,2 @@
+**/test/fixtures/**/node_modules/**
+**/test/work/*
```

Comment:
There is no technical reason to not support specifying ignore in the package.json, but eslint doesn't support it.

> The execution one is quite reasonable

For config file yes, but we specify config in the `package.json`